ts/index.ts

```diff
@@ -89,7 +89,8 @@ interface NativeMethods {
   ): Promise<Types.Profile>;
   deregisterUser(profileId: string): Promise<void>;
   handleRegistrationCallback(uri: string): Promise<void>;
-  cancelRegistration(): Promise<void>;
```

Comment:
Why do we delete generic `cancelRegistration`? I though we wanted to have one generic `cancel` that will cancel any ongoing registration?